client/src/lib/hooks/useConnection.ts

```diff
@@ -278,7 +279,9 @@ export function useConnection({
       setConnectionStatus("error-connecting-to-proxy");
       return;
     }
-    const mcpProxyServerUrl = new URL(`${getMCPProxyAddress(config)}/sse`);
+    const mcpProxyServerUrl = new URL(
+      `${config.MCP_PROXY_FULL_ADDRESS.value}/sse`,
```

Comment:
Now, this should probably use `/sse` or `/mcp` based on transport type.

Reply:
Has that changed in `main`? I got that update of the last merge and it was `/sse`